pkg/telemetry/prometheus/node.go

```diff
@@ -184,8 +194,13 @@ func GetUpdatedNodeStats(prev *livekit.NodeStats, prevAverage *livekit.NodeStats
 		stats.RetransmitPacketsOutPerSec = perSec(prevAverage.RetransmitPacketsOut, retransmitPacketsNow, elapsed)
 		stats.NackPerSec = perSec(prevAverage.NackTotal, nackTotalNow, elapsed)
 		stats.ParticipantJoinPerSec = perSec(prevAverage.ParticipantJoin, participantJoinNow, elapsed)
+		stats.ParticipantRtcPerSec = perSec(prevAverage.ParticipantRtc, participantRTCNow, elapsed)
 		stats.SysPacketsOutPerSec = perSec(uint64(prev.SysPacketsOut), uint64(sysPackets), elapsed)
 		stats.SysPacketsDroppedPerSec = perSec(uint64(prev.SysPacketsDropped), uint64(sysDroppedPackets), elapsed)
+		stats.TrackPublishAttemptsPerSec = perSec(uint64(prevAverage.NumTrackPublishAttempts), uint64(trackPublishAttemptsNow), elapsed)
+		stats.TrackPublishSuccessPerSec = perSec(uint64(prevAverage.NumTrackPublishSuccess), uint64(trackPublishSuccessNow), elapsed)
+		stats.TrackSubscribeAttemptsPerSec = perSec(uint64(prevAverage.NumTrackSubscribeAttempts), uint64(trackSubscribeAttemptsNow), elapsed)
+		stats.TrackSubscribeSuccessPerSec = perSec(uint64(prevAverage.NumTrackSubscribeSuccess), uint64(trackSubscribeSuccessNow), elapsed)
```

Comment:
Although I remember writing some bits to this code, I do not remember all details. A couple of comments here
1. These `perSec` things should be set in `stats` even when `computeAverage` is false. For example, `ParticipantJoinRatePerSec` is set in `stats` above and reset here if `computeAverage` is true.
2. For the average computation part, should it be using `prevAverage` (I know existing code uses `prevAverage` except for the `SysPacketsOut` and `SysPacketsDropped`) or just `prev`? I think `prevAverage` might be fine as it is only calculating rate here and the actual values in `prev` and `prevAverage` may be same? Anyhow, would be good to make this code all look the same. So, that might mean changing `SysPacketsOut` and `SysPacketsDropped` to use `prevAverage` too.

Reply:
Nice catch, updated these along with SysPacketsOut and SysPacketsDropped